src/modules/Snapper.rb

```diff
@@ -198,147 +337,97 @@ def ReadSnapshots
     end
 
 
-    def LastSnapperErrorMap
-      Convert.to_map(SCR.Read(path(".snapper.error")))
-    end
-
-
     def ReadConfigs
-      @configs = Convert.convert(
-        SCR.Read(path(".snapper.configs")),
-        :from => "any",
-        :to   => "list <string>"
-      )
 
-      if @configs == nil || @configs.empty?
-        return false
-      end
+      @configs = SnapperDbus.list_configs()
 
       if @configs.include?("root")
-        @current_config = "root"
+        self.current_config = "root"
+      elsif !@configs.empty?
+        self.current_config = @configs[0]
       else
-        @current_config = @configs[0]
+        self.current_config = ""
       end
 
-      return true
     end
 
 
-    # Initialize snapper agent
```

Comment:
Ok, maybe we're using it only in specs: https://github.com/yast/yast-yast2/commit/d9861081c77e3134d152ddfedf4ed3b7d1f6113d

Anyway, I'm using it on methods documentation just for coherence: https://github.com/yast/yast-yast2/blob/master/library/system/src/lib/yast2/fs_snapshot.rb.